Cauldron/Assets/Scripts/RecipeBook.cs

```diff
@@ -415,6 +428,12 @@ public void SwitchHighlight(RecipeBookEntry recipeBookEntry)
             CloseBook();
             tooltipManager.HighlightRecipe(recipeBookEntry.CurrentRecipe);
             OnSelectRecipe?.Invoke(recipeBookEntry.CurrentRecipe);
+            int eightyPercent = (int)((allMagicalRecipes.Count + allHerbalRecipes.Count) * 0.8);
+            if(PlayerPrefs.GetInt(PrefKeys.AutoCooking) == 0)
+                return;
+            
+            await UniTask.Delay(TimeSpan.FromSeconds(OpenCloseAnimationTime));
```

Comment:
Again, it's an async operation and those should generally support cancellation, or you should in some other way prevent user from clicking two potions one after another while the book is sliding upwards.